src/tests/catwalk_tests/test_storage.py

```diff
@@ -24,8 +25,6 @@ def __init__(self, val):
 
 def test_S3Store():
     with mock_s3():
-        import boto3
```

Comment:
cool

it looks like at this point you're fixing old stuff; but, if you like, while you're at it, I believe there's one more in that file.